.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,52 @@
+<!--
+If you want to report a bug, we added some points below you can fill out. If you want to request a feature, feel free to remove all the irrelevant text. In addition to this [issue tracker](https://github.com/diesel-rs/diesel/issues), you can also talk to Diesel maintainers and users on [Gitter](https://gitter.im/diesel-rs/diesel).
+-->
+
+## Setup
+
+### Versions
+
+- **Rust:**
+- **Diesel:**
+- **Database:**
+- **Feature Crates:**
```

Comment:
Well if you're using the chrono feature in diesel, the chrono version it's using is using is set by diesel.